assets/js/modules/analytics-4/components/settings/SettingsView.js

```diff
@@ -181,9 +215,52 @@ export default function SettingsView() {
 				</div>
 			</div>
 
-			<SettingsEnhancedMeasurementView />
-
 			<OptionalSettingsView />
+
+			<SettingsStatuses
+				statuses={
+					fpmEnabled
+						? [
+								{
+									label: __(
+										'Enhance Measurement',
+										'google-site-kit'
+									),
+									status: isEnhancedMeasurementStreamEnabled,
+								},
+								{
+									label: __(
+										'Enhanced Conversion Tracking',
+										'google-site-kit'
+									),
+									status: isConversionTrackingEnabled,
+								},
+								{
+									label: __(
+										'First-party Mode',
+										'google-site-kit'
+									),
+									status: isFirstPartyModeEnabled,
```

Comment:
@benbowler, we should check for the server requirements as well because if the server requirement checks fail, we still keep the `isEnabled` to `true` while updating the relevant properties and fallback the routing to the Google servers for measurements.

Reply:
cc: @techanvil

Reply:
We should update it for both the Analytics and Ads settings views.

Reply:
So it should show "Disabled" if any checks failed?

Reply:
That's right 👍